code/modules/mob/living/carbon/xenobiological/colours/colour_metal.dm

```diff
@@ -0,0 +1,19 @@
+/decl/slime_colour/metal
+	name = "metal"
+	descendants = list(
+		/decl/slime_colour/silver,
+		/decl/slime_colour/yellow,
+		/decl/slime_colour/gold,
+		/decl/slime_colour/gold
+	)
+	baby_icon =    'icons/mob/slimes/slime_baby_metal.dmi'
+	adult_icon =   'icons/mob/slimes/slime_adult_metal.dmi'
+	extract_icon = 'icons/mob/slimes/slime_extract_metal.dmi'
+	reaction_strings = list(/decl/material/solid/metal/uranium = "Synthesises steel and plasteel.")
+
+/decl/slime_colour/metal/handle_uranium_reaction(var/datum/reagents/holder)
+	var/obj/item/stack/material/steel/M = new (get_turf(holder.my_atom))
```

Comment:
can do new(loc,amount) for stacks instead of manually changing var, will handle icon updating etc that way